src/main/groovy/com/blackducksoftware/integration/hub/detect/hub/HubManager.java

```diff
@@ -150,6 +162,9 @@ public void performPostHubActions(final DetectProject detectProject, final Proje
             throw new DetectUserFriendlyException(e.getMessage(), e, ExitCodeType.FAILURE_HUB_CONNECTIVITY);
         } catch (final HubTimeoutExceededException e) {
             throw new DetectUserFriendlyException(e.getMessage(), e, ExitCodeType.FAILURE_TIMEOUT);
+        } catch (final DoesNotExistException e) {
+            throw new DetectUserFriendlyException(String.format("There was a problem unmapping/deleting old Code Locations: %s (this Code Location should have been created during this Detect run)", e.getMessage()), e,
```

Comment:
We are only unmapping code locations not deleting, so the text should be changed to reflect that

Reply:
Also, what does the text of the exception mean? We are not unmapping code locations from this run, we are unmapping code locations mapped to this version from before this run, right?